### ISSUE TITLE
Bug: Seed to Account

### ISSUE THREAD

**Issue by zgrguric** [AlexanderBuzz/xrpl-php#16]
Just used rippled wallet_propose to generate account:
Seed: ssNYVX6qYzKNu48FBBs4LuvgfivEJ
Account: rNWgYiADKLCJDfZX3oAdPcsRHe9vtJdCVD
key_type: secp256k1

![image](https://github.com/AlexanderBuzz/xrpl-php/assets/108837406/c6b96135-8bfd-4d40-a4eb-be9db794c8b6)

Using this I get wrong address, I think key type should be considered automatically?
```
$wallet = Wallet::fromSeed('ssNYVX6qYzKNu48FBBs4LuvgfivEJ');
echo 'Returned address: '.$wallet->getClassicAddress();
//Returned address:  rGkCCVz64f1UYVGK6Qpzd9e8J898Wtr9LY
```

**Comment by zgrguric** [comment]
I think it should throw error or atleast autodetect algo?

Not sure how xrpl.js do this.

**Comment by AlexanderBuzz** [comment]
It should, I'll have a look what is the issue here

**Comment by AlexanderBuzz** [comment]
@zgrguric So autodetection is implemented with test, release is v0.7.5. Thanks for battle testing this!
